src/haz3lweb/Grading.re

```diff
@@ -297,13 +302,25 @@ module ImplGradingReport = {
       ],
     );
 
-  let individual_reports = (~inject, ~hinted_results) =>
+  let individual_reports = (~inject, ~report) => {
+    let test_results =
+      switch (report.test_results) {
+      | Some(x) => x
+      | None => failwith("No test results")
```

Comment:
this shouldn't be a failwith when there are no tests -- seems to cause a crash when there are zero tests.

Reply:
fix